{
  "containers": {
    "adp": [
      {
        "providerMetadata": {
          "orgId": "af854a3a-2127-422b-91ae-364da2661108",
          "shortName": "CVE",
          "dateUpdated": "2024-08-04T01:51:20.273Z"
        },
        "title": "CVE Program Container",
        "references": [
          {
            "tags": [
              "x_refsource_CONFIRM",
              "x_transferred"
            ],
            "url": "https://www.ibm.com/support/pages/node/6586704"
          },
          {
            "name": "ibm-datapower-cve202138872-dos (208348)",
            "tags": [
              "vdb-entry",
              "x_refsource_XF",
              "x_transferred"
            ],
            "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/208348"
          }
        ]
      }
    ]
  },
  "cveMetadata": {
    "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
    "assignerShortName": "ibm",
    "cveId": "CVE-2021-38872",
    "datePublished": "2022-05-17T16:25:21.862308Z",
    "dateReserved": "2021-08-16T00:00:00",
    "dateUpdated": "2024-09-16T18:17:59.500Z",
    "state": "PUBLISHED"
  },
  "search": "解析验收测试文案"
}